### ISSUE TITLE
AI Assistant: Add toolbar option in paragraph block

### ISSUE THREAD

**Issue by renatoagds** [Automattic/jetpack#31321]
Add the toolbar option for one or multiple paragraph blocks.

This will include the dropdown list empty since they will be added in follow-up issues.

Depends on #31317

**Comment by ederrengifo** [comment]
These are the designs we defined in the latest iteration for the two use cases we should cover:

<img width="726" alt="jetpackai-gh-1" src="https://github.com/Automattic/jetpack/assets/10525007/2e7e94a4-0d75-4a32-95e1-ada1dab5789a">

We still need to confirm a couple of things: I'm still trying to figure out the tooltip's text, but I will update the design here if something changes. Another thing is the shortcut for the action, but I'm unsure if we want to address that as part of this issue.

**Comment by keoshi** [comment]
> I'm still trying to figure out the tooltip's text

@ederrengifo Since this one is contextual, what do you think about: “Show AI Assistant options” or simply “AI Assistant options”.

For the shortcut it would be nice to have something that's widely usable across multiple keyboard layouts but also something basic, like `Tab + A` for **A**I or **A**ssistant.
